### DISCUSSION TITLE
2023-10-26 SIG-infra kick off

### DISCUSSION THREAD

**Discussion by myitcv** [cue-lang/cue#2659] [Community Calls]
Per @verdverm [on Slack](https://cuelang.slack.com/archives/CRJRL2X7F/p1698172807734119):

> I've set the first meeting for Thursday Oct 26 1500 UTC (the same time CUE modules feedback call typically happen)
> 
> * [discussion slides] https://docs.google.com/presentation/d/1VDlFfnI8vuV3I-yaPiZ3tnLDcRaUDnh3pCnCgOXOr74/edit?usp=sharing
> * [google meet link] https://meet.google.com/eai-agjw-nby
> 
> I've broadened the name to "SIG-infra" for now, since we are all probably interested in these things and not just TF.
> 
> If you want to add a slide(s), DM me and I can give you editor access to the slides, or you can just tell me to add an item so we don't forget to talk about it.

**Comment by dominikdm** [comment]
The recording of the video can be found here: https://youtu.be/AIqOKiVQJjk
